source/content/visual-studio-code.md

```diff
@@ -44,25 +46,25 @@ wp-content/upgrade/
 
 ```
 
-At this point, you should see your site files in the File Explorer on the left side of VS Code, and you can open any of these for editing in the main window by clicking on the file name.
+At this point, you should see your site files in the File Explorer on the left side of VS Code. Click the file name to open it for editing in the main window.
 
-## Deploy your local code changes to a Pantheon environment
+## Deploy local code changes to a Pantheon environment
 
 Once you've made some code changes, you'll want to push those up to your Pantheon Dev or Multidev environment. You can do this using Git, or with SFTP.
 
-### Push your changes with Git (recommended)
+### Push changes with Git (recommended)
 VS Code includes Git integration and an integrated terminal, either of which can be used to make Git changes and push them back to your Pantheon repository and Dev environment.
 
-Make sure your site is in [Git mode](//guides/quickstart/connection-modes) before pushing code via Git.
+Make sure your site is in [Git mode](/guides/quickstart/connection-modes) before pushing code via Git.
 
 #### [Git Version Control in VS Code](https://code.visualstudio.com/docs/introvideos/versioncontrol)
```

Comment:
With the link added on L61 I'd say we take it out of the H3, and do the same for the next section.

Reply:
Good call, agreed